pages/_app.tsx

```diff
@@ -41,17 +43,30 @@ export default function App({ Component, pageProps }) {
 
   React.useEffect(() => {
     function onRouteChangeComplete() {
-      Fathom.trackPageview()
+      if (fathomId) {
+        Fathom.trackPageview()
+      }
+      if (postHogId) {
+        // See https://posthog.com/docs/integrate/client/js#one-page-apps-and-page-views
```

Comment:
let's remove the extra comments. they're not adding much, the URLs may change over time, and devs can just google posthog if they want to get more info

Reply:
I find it easier to look up information when it's documented this way, but I'll remove it. Also, it pointed to a specific section about one page apps which requires special care.